space.fips.Fips.json

```diff
@@ -0,0 +1,42 @@
+{
+	"app-id": "space.fips.Fips",
```

Comment:
The app-id is wrong. It should be something like `fips.space.Fips` as you don't own the `space.fips` domain name.

Reply:
Hi, this one I didn't get.
I see an example here http://docs.flatpak.org/en/latest/first-build.html#add-a-manifest
and I see reverse-DNS app-id (https://dbus.freedesktop.org/doc/dbus-specification.html#message-protocol-names)
```
{
    "app-id": "org.flatpak.Hello",
    "runtime": "org.freedesktop.Platform",
```

So, I do own http://fips.space domain, I suppose the reverse would be `space.fips`, no?

Reply:
Yes, sorry :P